extensions/git/src/timelineProvider.ts

```diff
@@ -200,7 +200,7 @@ export class GitTimelineProvider implements TimelineProvider {
 			if (working) {
 				const date = new Date();
 
-				const item = new GitTimelineItem('', index ? '~' : 'HEAD', localize('git.timeline.uncommitedChanges', 'Uncommited Changes'), date.getTime(), 'working', 'git:file:working');
+				const item = new GitTimelineItem('', index ? '~' : 'HEAD', localize('git.timeline.uncommitedChanges', 'Uncommitted Changes'), date.getTime(), 'working', 'git:file:working');
```

Comment:
```suggestion
				const item = new GitTimelineItem('', index ? '~' : 'HEAD', localize('git.timeline.uncommittedChanges', 'Uncommitted Changes'), date.getTime(), 'working', 'git:file:working');
```

Reply:
I skipped the localization key with the assumption that translations (which are external) would need to also be updated.